### DISCUSSION TITLE
Resource loop stopping progression

### DISCUSSION THREAD

**Discussion by MisterCadrega** [TechReborn/TechReborn#3093] [General]
So, I see you need Nickel (for the cupronickel coil) to make the Industrial Blast Furnace.
To make nickel, you need the industrial grinder so you can obtain it from common ores (cut using the Rock cutter or anything with Silk Touch)
To make the industrial grinder, you need Steel ingots, that can be obtain only from the Blast furnace....
To make the blast furnace, you need Nickel....and the loop begins again.

Am I missing something?

**Comment by Ayutac** [comment]
you get nickel from centrifuging copper, iron or gold

**Reply by MisterCadrega** [reply]
Ah, weird, the recipe is not showing up on REI
